src/orchard/inspect.clj

```diff
@@ -686,7 +689,7 @@
 
 (defn- render-section [obj inspector [section sort-key-fn]]
   (let [method (symbol (str ".get" (name section)))
-        elements (eval (list method obj))]
+        elements (eval (list method obj))] ;; XXX ditch eval
```

Comment:
XXX? Should be `TODO:` or `FIXME:`?

Reply:
> XXX? Should be TODO: or FIXME:?

fixed by actually addressing the thing 😄